compose/material3/material3/src/skikoMain/kotlin/androidx/compose/material3/ModalBottomSheet.skiko.kt

```diff
@@ -31,9 +38,17 @@ internal actual fun ModalBottomSheetPopup(
     content: @Composable () -> Unit,
 ) {
     Popup(
+        popupPositionProvider = object : PopupPositionProvider {
+            override fun calculatePosition(
+                anchorBounds: IntRect,
+                windowSize: IntSize,
+                layoutDirection: LayoutDirection,
+                popupContentSize: IntSize
+            ) = IntOffset.Zero
+        },
         onDismissRequest = onDismissRequest
     ) {
-        Box(Modifier.windowInsetsPadding(windowInsets)) {
+        Box(Modifier.windowInsetsPadding(windowInsets.exclude(WindowInsets.systemBars))) {
```

Comment:
It overrides the default value `BottomSheetDefaults.windowInsets get() = WindowInsets.systemBars.only(WindowInsetsSides.Vertical)`.

It seems it was an intention to use `WindowInsets.systemBars` for insets 🤔. But I can't understand this intention. How `ModalBottomSheet` look on Android in a fullscreen application with default arguments?

Reply:
> How ModalBottomSheet look on Android in a fullscreen application with default arguments?

See my video in this comment - https://github.com/JetBrains/compose-multiplatform/issues/3701#issuecomment-1723285883

Reply:
After thinking, I can reason about why we should add insets padding - to be able to press bottom items. But I think, better to add a spacer item than move the sheet enterely.

Anyway, it seems it won't do anything, because we apply insets on the Popup level? Or we apply padding 2 times without this change? If we apply it 2 times - is it by design, or an iOS implementation bug?

Reply:
Yep, twice. Implementation bug, see screenshots above